docs/concepts/governance/02-process.md

```diff
@@ -78,7 +78,7 @@ To create an onchain Governance Proposal:
 
 2. Create a topic in the [Governance Forum](https://gov.uniswap.org/) titled "Governance Proposal — [Your Title Here]" and link to previous forum posts and the Temperature Check Snapshot poll.
 
-3. Create your proposal. This can be done either through an interface (e.g. [Tally](https://tally.xyz/gov/uniswap)) or through writing the calldata for more complicated proposal logic. If the proposal passed, this calldata will execute. If writing the calldata yourself, please review the logic with a qualified Uniswap community member prior to posting the proposal.
+3. Create your proposal. This can be done either through an interface (e.g. [Tally](https://tally.xyz/gov/uniswap)) or through writing the calldata for more complicated proposal logic. If the proposal passed, this calldata will execute. If writing the calldata yourself, please review the logic with a qualified Uniswap community member prior to posting the proposal. For more information on developing and advancing multi-chain Uniswap Governance Proposals, see [Multi-Chain Governance](../multi-chain-proposals.md).
```

Comment:
[arbitrary comment location]

at the top of this doc, it gives the "last updated" date. I don't love this convention but to be consistent we should change it